src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new Random().nextInt(100), new ColorSupplier().getRandomColor());
```

Comment:
make global Random and ColorSupplier fields and use them here

Reply:
Why is your choice better?